### ISSUE TITLE
Model gives large error on evaluation set. 

### ISSUE THREAD

**Issue by neonithinar** [neonithinar/pycharm_projects#1]
I am trying to learn Deep learning. the code given in the files cats and dogs.py, cifar10 model, etc are the same implementations from various tutorials and videos. Even Though I have used early stopping callbacks the model seems to perform so worse on model.evaluate() method. I confirmed the same issue with my MNIST training model which gave 98% validation accuracy on training. The model was only able to give a 9.4% accuracy with model.evaluate method with the same validation set.
